docs/clients/p4rt-ctl.rst

```diff
@@ -91,6 +91,24 @@ services that are available for a p4runtime client.
 For information on security and enabling TLS on gRPC ports, refer to
 guides/security-guide.md
 
+Usage: p4rt-ctl [OPTIONS] COMMAND [ARG...]
+positional arguments:
+  command               Subcommand to run
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -g GRPC_ADDR, --grpc_addr GRPC_ADDR P4Runtime gRPC server address, format : <server IP>:<port>
+
+Details about supported option (``-g`` GRPC_ADDR)::
+---------------------------------------------------
+This is an optional parameter and default value is 127.0.0.1:9559. If P4Runtime server is not running
+locally, user can specify the address using this new option.
```

Comment:
Just use "using this option" instead of "using this new option" (since "new" can be outdated in time and this will stay in documentation)